ext-src/stubs/php_swoole_lock.stub.php

```diff
@@ -6,7 +6,7 @@ public function __destruct() {}
         public function lock(): bool {}
         public function locakwait(float $timeout = 1.0): bool {}
         public function trylock(): bool {}
-        public function lock_read(): bool {}
+        public function lock_read(?float $timeout = null): bool {}
```

Comment:
Do not modify lock_rd() method, but modify lock_wait(), add an int type parameter, and specify whether the lock type is wrlock or rdlock.